src/routes/login.svelte

```diff
@@ -26,7 +26,7 @@
     const privateCode = urlParams.get('privateCode');
 
     const base64 = btoa(location.host + '/login');
-    link = `https://fluffyscratch.hampton.pw/auth/getKeys/v2?redirect=${base64}`;
+    link = `https://auth.itinerary.eu.org/auth/?redirect=${base64}`;
```

Comment:
The `name` parameter goes here: `https://auth.itinerary.eu.org/auth/?redirect=${base64}&name=Aviate`